src/styles/partials/components/_summary-list.scss

```diff
@@ -38,6 +38,8 @@
     text-align: center;
     font-size: $h1-font-size;
     margin-right: $default-padding;
+    width: 45px;
+    max-width: 45px;
   }
```

Comment:
would these be $padding-largest?

Reply:
Wanted to discuss this one before making the update. The positive to this, it would be it makes the update pretty simple by using a variable that can be updated from one location in this instance. One negative I can see to updating a width property with the padding variable, is if there is ever a change to padding-largest, we would need to go back and update the unit for the width. That might not be bad in one instance, but if it opens the door to using spacing variables in other width instances, across multiple files, that could become tedious down the road. Thoughts?

Reply:
@theilman7 maybe we need width variables? Which might be biting off more than we can chew at the moment. So yeah I think leave it as is and we can revisit width variables later.